scripts/SANS/ISISCommandInterface.py

```diff
@@ -494,107 +494,26 @@ def WavRangeReduction(wav_start=None, wav_end=None, full_trans_wav=None, name_su
 
         ReductionSingleton().instrument.setDetector('front')
 
-        retWSname_front = _WavRangeReduction(name_suffix)
+        retWSname_front, front_slices = _WavRangeReduction(name_suffix)
         retWSname = retWSname_front
 
     # This section provides a the REAR -- FRONT fitting and a stitched workspace.
     # If merge_flag is selected we use SANSStitch and get the fitting for free
     # If fitRequired is selected, then we explicity call the SANSFitScale algorithm
     if merge_flag:
-        # Prepare the Norm and Count workspaces for the FRONT and the REAR detectors
-        retWSname_merged = retWSname_rear
```

Comment:
It looks like `retWSname_merged` might still be used further on in the function even though it's no longer being set in this scope?

Reply:
I think the naming in the new GUI is wrong here but this is unrelated to these changes which all relate to the old GUI. I've corrected the fact that resWSname_merged was not being used as well.